apps/condo/domains/acquiring/schema/RegisterMultiPaymentForInvoicesService.test.js

```diff
@@ -336,6 +343,72 @@ describe('RegisterMultiPaymentForInvoicesService', () => {
                 }),
             ]))
         })
+
+        test('register multiPayment for invoice from different organization', async () => {
+            const utils1 = new TestUtils([ResidentTestMixin])
+            const utils2 = new TestUtils([ResidentTestMixin])
+            await utils1.init()
+            await utils2.init()
+
+            await utils2.updateAcquiringContext({
+                invoiceStatus: CONTEXT_FINISHED_STATUS,
+                invoiceImplicitFeeDistributionSchema: [{
+                    recipient: 'organization',
+                    percent: '5',
+                }],
+                invoiceRecipient: createTestRecipient(),
+            })
+
+            const resident = await utils1.createResident()
+
+            const [marketCategory] = await createTestMarketCategory(utils2.clients.admin)
+            const [marketItem] = await createTestMarketItem(utils2.clients.admin, marketCategory, utils2.organization)
+            const [itemPrice] = await createTestMarketItemPrice(utils2.clients.admin, marketItem)
+            const [priceScope] = await createTestMarketPriceScope(utils2.clients.admin, itemPrice, utils2.property)
+
+            let [invoice] = await registerResidentInvoiceByTestClient(
+                utils1.clients.resident,
+                pick(resident, 'id'),
+                [{
+                    priceScope: pick(priceScope, 'id'),
+                    count: 1,
+                }],
+            );
+
+            [invoice] = await updateTestInvoice(utils2.clients.admin, invoice.id, { status: INVOICE_STATUS_PUBLISHED })
+
+            const [result] = await registerMultiPaymentForInvoicesByTestClient(adminClient, {
+                invoices: [pick(invoice, 'id')],
+            })
+
+            expect(result).toBeDefined()
+            expect(result).toHaveProperty('multiPaymentId')
+
+            const multiPaymentId = result.multiPaymentId
+            const hostUrl = utils2.acquiringIntegration.hostUrl
+            expect(result).toMatchObject({
+                dv: 1,
+                webViewUrl: `${hostUrl}/pay/${multiPaymentId}`,
+                feeCalculationUrl: `${hostUrl}/api/fee/${multiPaymentId}`,
+                directPaymentUrl: `${hostUrl}/api/pay/${multiPaymentId}`,
+                anonymousPaymentUrl: `${hostUrl}/api/anonymous/pay/${multiPaymentId}`,
+            })
+
+            const multiPayment = await MultiPayment.getOne(adminClient, { id: multiPaymentId })
```

Comment:
we are trying to avoid using admin client (and btw, you can get admin client from utils, not from globals)